### PR TITLE
Create Zapier Section (and fixes)

### PR DESCRIPTION
**Pull request by sammullican**
Creates Zapier section under CRM Integration section, and fixes format of "LOCALiQ" across the documentation.

### CODE REVIEW

File: source/includes/_zapier.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- The LOCALiQ Zapier® app can be used at no additional cost.
+- Zapier® offers a free trial if you would like to test this solution.
+- The setup can be be completed by client or CSM.
+
+NOTE: There is an additional monthly subscription cost through Zapier® The cost depends on the number of leads sent per month and the number of leads transmitted between the systems (which Zapier® refers to as "Tasks"). Pricing information for Zapier® can be found [here](https://zapier.com/pricing).
+
+**Parameters:**
+
+With the LOCALiQ App you can get contacts and contacts interactions for ingestion into the CRM of your choice. The CRM should also have an app available on Zapier®.
```
Comment (clarkbremer) [inline source/includes/_zapier.md]: Replace "should" with "must" ?

Comment (clarkbremer) [inline source/includes/_zapier.md]: replace "contacts interactions" with "contact interactions"

File: source/includes/_zapier.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Contacts: This HIPAA Compliant endpoint supports integration with CRM products. It returns all contacts (first name, last name, email and/or phone number) generated by a campaign for an advertiser. The response is organized by campaigns with the contacts nested within. Please note that the response also includes the associated contact and its meta data.
```
Comment (clarkbremer) [inline source/includes/_zapier.md]: I don't think this is quite what the webhook returns.  It delivers them one at a time as they are created in near-real-time.  I think you may be confusing the payload with what's returned in the Contacts API call.

File: source/includes/_zapier.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Contact Interactions: This HIPAA Compliant endpoint supports integration with CRM products. It returns all interactions (or leads) generated by a campaign for an advertiser. The response is organized by campaigns with the interactions nested within. Please note that the response also includes the associated contact and its meta data.
```
Comment (clarkbremer) [inline source/includes/_zapier.md]: Similar to above

File: source/includes/_zapier.md (hunk 1 above)
Comment (sammullican) [reply source/includes/_zapier.md]: This has been updated to "must".

Comment (sammullican) [reply source/includes/_zapier.md]: This has been updated to "contact interactions".

File: source/includes/_zapier.md (hunk 2 above)
Comment (sammullican) [reply source/includes/_zapier.md]: I had copied this from the Contacts API/Contact Interactions API sections per Jayanthi's instructions, but I agree that it didn't apply here. I simplified it based on your comment, but let me know if you want this section revised.

File: source/includes/_zapier.md (hunk 3 above)
Comment (sammullican) [reply source/includes/_zapier.md]: Similar to above

File: source/includes/_zapier.md [hunk 4]
```diff
@@ -10,19 +10,19 @@ NOTE: There is an additional monthly subscription cost through Zapier® The cost
 
 **Parameters:**
 
-With the LOCALiQ App you can get contacts and contacts interactions for ingestion into the CRM of your choice. The CRM should also have an app available on Zapier®.
+With the LOCALiQ App you can get contacts and contact interactions for ingestion into the CRM of your choice. The CRM must also have an app available on Zapier®.
 
-Contacts: This HIPAA Compliant endpoint supports integration with CRM products. It returns all contacts (first name, last name, email and/or phone number) generated by a campaign for an advertiser. The response is organized by campaigns with the contacts nested within. Please note that the response also includes the associated contact and its meta data.
+Contacts: This HIPAA Compliant endpoint supports integration with CRM products. It delivers contact data (first name, last name, email and/or phone number) generated for an advertiser in near real-time.
```
Comment (clarkbremer) [inline source/includes/_zapier.md]: Replace "This HIPAA Compliant endpoint" with "This trigger".  I don't think it's HIPAA compliant.  
Add "etc" to list of fields.

File: source/includes/_zapier.md [hunk 5]
```diff
@@ -10,19 +10,19 @@ NOTE: There is an additional monthly subscription cost through Zapier® The cost
...
 
-Contact Interactions: This HIPAA Compliant endpoint supports integration with CRM products. It returns all interactions (or leads) generated by a campaign for an advertiser. The response is organized by campaigns with the interactions nested within. Please note that the response also includes the associated contact and its meta data.
+Contact Interactions: This HIPAA Compliant endpoint supports integration with CRM products. It delivers all interactions (or leads) generated for an advertiser in near real-time.
```
Comment (clarkbremer) [inline source/includes/_zapier.md]: Similar

File: source/includes/_zapier.md (hunk 4 above)
Comment (sammullican) [reply source/includes/_zapier.md]: Updated to "This trigger..." and added "etc" to list of fields.

File: source/includes/_zapier.md (hunk 5 above)
Comment (sammullican) [reply source/includes/_zapier.md]: Updated to "This trigger..."

**Pull request merged**